keyboards/orthocode/rules.mk

```diff
@@ -11,7 +11,7 @@ BOOTLOADER = usbasploader
 #   change yes to no to disable
 #
 BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = yes       # Mouse keys
+MOUSEKEY_ENABLE = no       # Mouse keys
```

Comment:
It may be better to enable LTO, as that will reduce firmware size.

Reply:
On compile the firmware size is 19402/28672, 67%, LTO changes it to 16700/28672 58%. 
I'm not too familiar with the impact of lto, much less how it interacts with via, I understand it breaks tmk layers and macros which I dont think affects via but again I'm not sure. Do you think it will affect it?

Reply:
It has no issues with VIA, and is enabled on a majority of the AVR (atmega32u4) based boards with VIA enabled on them, due to size constraints (part of why I recommended it). 

It doesn't break layers, but it does break action functions and action macros (eg, legacy TMK features, which are in the process of being removed, actually).  And `LTO_ENABLE` actually disables those functions because it does break them.

Additionally, LTO Can cause some issues with timing related stuff. But most of that has been handled by disabling optimization for files effected.  So, it should be very safe to use LTO.

Reply:
Added in lto the the main rules.mk and re-enabled mousekeys

Reply:
I can't test these changes anymore because atmega32a boards are not building on main with a fresh repo download.
An "unknown MCU 'atmega32a' specified" error is thrown.